modules/analytics-v2/src/views/full/index.tsx

```diff
@@ -149,49 +207,58 @@ export default class Analytics extends React.Component<{ bp: any }> {
     const received = this.getMetricCount('msg_received_count')
     const none = this.getMetricCount('msg_nlu_none')
     const percent = ((received - none) / received) * 100
-    return percent.toFixed(2) + '%'
+    return `${this.getNotNaN(percent)}%`
   }
 
   getTopLevelUnderstoodPercent = () => {
     const received = this.getMetricCount('msg_received_count')
     const none = this.getMetricCount('top_msg_nlu_none')
     const percent = ((received - none) / received) * 100
-    return percent.toFixed(2) + '%'
+    return `${this.getNotNaN(percent)}%`
   }
 
   getReturningUsers = () => {
     const activeUsersCount = this.getMetricCount('active_users_count')
     const newUsersCount = this.getMetricCount('new_users_count')
-    return ((newUsersCount / activeUsersCount) * 100).toFixed(2) + '%'
+    const percent = (newUsersCount / activeUsersCount) * 100
+    return `${this.getNotNaN(percent)}%`
   }
 
+  getNotNaN = value => (Number.isNaN(value) ? 0 : value.toFixed(2))
```

Comment:
I think that 0 can be misleading  (if there's an error fetching results, it doesn't necessarily means that the value is 0 - which may hide an issue). Usually I like to replace this with N/A instead